dracut/30afterburn/module-setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+check() {
+    return 0
+}
+
+depends() {
+    echo systemd network
+}
+
+install() {
+    inst_multiple afterburn
+
+    inst_simple "$moddir/afterburn-hostname.service" \
+        "$systemdutildir/system/afterburn-hostname.service"
+
+    # We want the afterburn-hostname to be firstboot only, so ignition-provided
+    # hostname changes do not get overwritten on subsequentboots
```

Comment:
Minor: `subsequent boots`
Super minor: `Ignition`

Reply:
Fixed up, thanks for the review!